packages/store/package.json

```diff
@@ -5,7 +5,12 @@
   "sideEffects": false,
   "peerDependencies": {
     "@angular/core": ">=19.0.0 <20.0.0",
-    "rxjs": ">=7.0.0"
+    "rxjs": ">=7.0.0",
+    "ts-morph": "21.0.1"
```

Comment:
@arturovt is it OK to have ts-moprh as a peer dep or would it be better to get rid of this dependency completely?

Reply:
I think it's better to remove it as I'm pretty sure people wouldn't be happy having it as a peer.

Reply:
I removed the ts-morph